bin/init.sh

```diff
@@ -56,7 +56,33 @@ if [ ! -f vendor/envoy-$PROXYVERSION ] ; then
     mkdir -p $OUT
     pushd $OUT
     # New version of envoy downloaded. Save it to cache, and clean any old version.
-    curl -Lo - https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xz
+
+    DOWNLOAD_COMMAND=""
+    if command -v curl > /dev/null; then
```

Comment:
I've never heard of 'command' before, I think it would work the same without it ?

Reply:
Here it is http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html, POSIX standard.

I found it in this stackoverflow answer https://stackoverflow.com/a/677212/6028420.